ethereum/parsers.schema.json

```diff
@@ -174,7 +174,10 @@
                                         "claim_reward",
                                         "configure",
                                         "buy",
-                                        "approve"
+                                        "approve",
+                                        "read",
```

Comment:
"read" is not present in PR adding new function type in PSM
https://github.com/LedgerHQ/python-smart-contract/pull/56/files